src/main/java/gov/usgs/ngwmn/model/MediationType.java

```diff
@@ -24,4 +25,27 @@ public static String validMediations() {
 		}
 		return validValues;
 	}
+	
+	/**
+	 * AboveDatum and Ascending are equal sort
+	 * BelowLand and Descending are equal sort
+	 * @param obj instance to compare
+	 * @return true if equal
+	 */
+	public boolean equal(Object obj) {
```

Comment:
Ah, answering my own questions as I go.  This is our method, no need to go with `.equals` like I asked above.

Reply:
I had the idea in the back of my mind to change the method name to reflect the sort. I think the name is better now.